syntax = "proto3";

service Reporter {
  rpc WeatherReport(WeatherRequest) returns (WeatherResponse) {}
}

message WeatherRequest {
  // Add fields for the request message here
}

message WeatherResponse {
  string json_message = 1;
}
